### ISSUE TITLE
Radeon pro V520 error not suported

### ISSUE THREAD

**Issue by andremxmx** [Lolliedieb/lolMiner-releases#894]
Setup Miner...
OpenCL driver detected.
Number of OpenCL supported GPUs: 1
No Cuda driver or GPUs detected.
Device 0:
    Name:    AMD Radeon Pro V520 MxGPU
    Address: 0:30
    Vendor:  Advanced Micro Devices (AMD)
    Drivers: OpenCL
    Memory:  8048 MByte (7984 MByte free)
    Active:  false (Unsupported device or driver version.)

[https://i.ibb.co/XD40H44/Screenshot-1.png](url)

**Comment by jgonzis** [comment]
That should be added in the near future.

**Comment by wes2021** [comment]
andremxmx I'm having the same issue. Were you able to resolve?

**Comment by andremxmx** [comment]
Yes this was solve whit a beta version but the problem is only get 2.8 mh so low, i just waiting for Lolliedieb
 to make an update after new year!

**Comment by andremxmx** [comment]
![98ef4362-3a87-4268-9a10-b62bec36917c](https://user-images.githubusercontent.com/10226983/147757690-92c709e8-16f2-4720-99f4-8ce81b930e23.jpg)

**Comment by andremxmx** [comment]
[lolMiner_v1.39_alpha2_Lin64.tar.gz](https://github.com/Lolliedieb/lolMiner-releases/files/7793172/lolMiner_v1.39_alpha2_Lin64.tar.gz)

**Comment by jgonzis** [comment]
> [lolMiner_v1.39_alpha2_Lin64.tar.gz](https://github.com/Lolliedieb/lolMiner-releases/files/7793172/lolMiner_v1.39_alpha2_Lin64.tar.gz)

that seems is not applying the OC that is running like in standby...

**Comment by andremxmx** [comment]
> andremxmx I'm having the same issue. Were you able to resolve?

I fix it whit another driver, what you need to do:



Ubuntu 20 mandatory

apt-get update -y

wget http://repo.radeon.com/amdgpu-install/21.40.2/ubuntu/focal/amdgpu-install_21.40.2.40502-1_all.deb

dpkg -i amdgpu-install_21.40.2.40502-1_all.deb

apt --fix-broken install -y

apt-get update

reboot  

dpkg -i amdgpu-install_21.40.2.40502-1_all.deb

apt-get update

reboot

amdgpu-install --no-32 -y

reboot

install the miner and run!

you will get 48 - 50 mh/s in each GPU OC and you can get 60 mh/s at max
